### ISSUE TITLE
Question: One process per FMU

### ISSUE THREAD

**Issue by jdsika** [eclipse/openmcx#10]
Hi team,

I wanted to ask if OpenMCX is spinning up a single process for every FMU or if everything is handled in a single process?

Best regards
Carlo

**Comment by jdsika** [comment]
Thx! Is this somewhere part of the documentation?

**Comment by ClemensLinnhoff** [comment]
The stepType is an OpenMCx proprietary parameter, that is not part of the SSP standard. Therefore, it should be documented in this repository. While I was able to find the different options for the stepType in the [task schema file](https://github.com/eclipse/openmcx/blob/main/scripts/SSP/Task.xsd), I could not find further explanation on what the different options do.
It would be nice, if further documentation about this could be added to the readme or a separate documentation file.

**Comment by ClemensLinnhoff** [comment]
Furthermore, I did some tests with two sensor models connected to the same input, a SensorView from esmini, as shown in the image below.

![2SensorSetup](https://github.com/eclipse/openmcx/assets/27010086/287f6908-e586-44bb-97be-fc3e97ba35ce)

However, I could not make out any change in execution speed. I tested sequential, parallel_one_step_size, parallel_sync_all and for all three the difference in execution time was only a few milliseconds.
